src/recipe_dictionary.cpp

```diff
@@ -32,10 +32,121 @@ const recipe &recipe_dictionary::get_uncraft( const itype_id &id )
 std::vector<const recipe *> recipe_subset::search( const std::string &txt ) const
 {
     std::vector<const recipe *> res;
-
+    std::vector<std::string> subqueries;
+    std::vector<std::string> requirement_qualities,
```

Comment:
I'd repeat the definition here and then add a line break after this section. I don't think the continuation declaration adds much in terms of clarity